README.md

```diff
@@ -44,3 +44,29 @@ from the MSC.
 * `./scripts/build.sh`
 * `./dist/bin`
 * Access at <http://localhost:8080>
+
+## Docker
+
+### Building
+
+In order to build a docker image, run the following in the root directory:
+
+`$ docker build . -t matrix/materwall`
```

Comment:
Why `matrix/`? Perhaps I am missing some docker knowledge here?

Reply:
It's just a tag, you can use any tag you want locally 🙂 I just scoped it for `matrix` to avoid confusion (in case there is another waterfall on a system 🤔) + it played well with the path to the `waterfall` in our container registry, so I thought for consistency it would be less confusing.